### PR TITLE
Task schema

### PR DESCRIPTION
**Pull request by meganhng**

### CODE REVIEW

**Comment by AnsonTu** [comment]
What are the changes to the yarn.lock, package.json, and package-lock.json files for? It looks like you're adding yarn to them?

File: client/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "redux": "^4.0.4",
     "redux-form": "^8.2.6",
     "redux-thunk": "^2.3.0",
+    "styled-components": "^4.4.1"
```
Comment (AnsonTu) [inline client/package.json]: This actually breaks the code, since it's missing a comma... 
How did you add this line here? Was this part of your rebase?

Comment (meganhng) [reply client/package.json]: Oh i really quickly changed it on github, I meant to delete the yarn thing

**Comment by meganhng** [comment]
This got all weird in rebase; closing for now

**Pull request closed without merging**